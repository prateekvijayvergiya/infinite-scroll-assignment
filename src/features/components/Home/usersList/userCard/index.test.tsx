import Component from './'
import makeSnapshot from './../../../../testing/makeSnapshot';

jest.mock('@material-ui/core/Card', () => 'Card')
jest.mock('@material-ui/core/CardActionArea', () => 'CardActionArea')
jest.mock('@material-ui/core/CardContent', () => 'CardContent')
jest.mock('@material-ui/core/CardMedia', () => 'CardMedia')
jest.mock('@material-ui/core/Typography', () => 'Typography')

describe('UserCard', () => {
  it('renders correctly', () => {
    const props = {
      id:'1',
      firstName: 'Alan',
      lastName: 'Walker',
      picture: 'test'
    }
    makeSnapshot(<Component {...props}/>)
  })
})
