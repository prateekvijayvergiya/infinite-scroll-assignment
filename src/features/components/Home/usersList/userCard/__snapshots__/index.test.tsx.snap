// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`UserCard renders correctly 1`] = `
<Card
  className="makeStyles-root"
>
  <CardActionArea>
    <CardMedia
      alt="user image"
      component="img"
      height="200"
      image="test"
    />
    <CardContent>
      <Typography
        className="makeStyles-text"
        gutterBottom={true}
        variant="h5"
      >
        Alan
         
        Walker
      </Typography>
    </CardContent>
  </CardActionArea>
</Card>
`;
